samples/testing/dump_operation/test_dump_operation.py

```diff
@@ -0,0 +1,119 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import qsharp
+from qsharp.utils import dump_operation
+
+def test_empty_operation() -> None:
+    qsharp.init(target_profile=qsharp.TargetProfile.Unrestricted)
+    
+    res = dump_operation("qs => ()", 1)
+    assert res == [
+        [complex(1.0, 0.0), complex(0.0, 0.0)],
```

Comment:
Do you need to call complex explicitly, or can you compare the numbers with just [1, 0], [0, 1]? I think Python is nice enough about type casting to allow to compare floating point with complex directly, I do that in my tests